spec/descriptions/tagSyntheticSettings.md

```diff
@@ -8,10 +8,10 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
 - **playbackCapability** The playback capabilities provided by this location resource.
   The playbackCapability object has the following properties: 
   - **syntheticType** Different types of synthetic tests that can be executed at this location. 
-    Possible values are HTTPAction, HTTPScript, BrowserScript, WebpageAction, WebpageScript, and DNSAction. 
```

Comment:
I would suggest to remove all words around DNSAction and DNSActionConfiguration since we do not support now.

Reply:
Have removed DNSAction from the file.